corehq/apps/app_manager/templates/app_manager/partials/forms/case_config_ko_templates.html

```diff
@@ -213,9 +213,18 @@ <h4 class="panel-title panel-title-nolink">
           {% trans "Question" %}
         </th>
         <th class="col-sm-1"></th>
-        <th class="col-sm-5">
+        <th class="col-sm-3">
           {% trans "Case Property" %}
         </th>
+        <th class="col-sm-1 text-center text-nowrap" style="position:relative">
+            <span style="padding-right: 25px">{% trans "Save Only If Edited" %}</span>
+            {% comment %} These odd style additions were necessary to get the question mark icon to
```

Comment:
Agreed. I was taught inline styling is taboo, but when it's hacky like this and has a weird effect on the layout I prefer it to be clearly visible in the code (and in a browser's "inspect") instead of hidden behind a class in another file.